### PR TITLE
remove docker version spec in ci

### PR DESCRIPTION
**Pull request by armaanv**
use the default docker version rather than specifying one per job.

### CODE REVIEW

**Review by molinamelendezj — APPROVED** [review]
we will always get the default version then?

File: etc/testing/circle/install.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ fi
 # To get the latest minikube version:
 # curl https://api.github.com/repos/kubernetes/minikube/releases | jq -r .[].tag_name | sort -V | tail -n1
 if [ ! -f cached-deps/minikube ] ; then
-    MINIKUBE_VERSION=v1.26.0 
+    MINIKUBE_VERSION=v1.27.0
```
Comment (chainlink) [inline etc/testing/circle/install.sh:38]: We've had some issues with caching here, we should make sure 1.27 is actually getting installed.

**Review by chainlink — APPROVED** [review]
one small comment

**Comment by msteffen** [comment]
Closing this PR, as it's obsolete.

**Pull request closed without merging**